vimeo-networking/src/main/java/com/vimeo/networking/VimeoClient.java

```diff
@@ -221,23 +222,14 @@ public void setVimeoAccount(@Nullable VimeoAccount vimeoAccount) {
     }
 
     /**
-     * @deprecated use {@link #saveAccount(VimeoAccount, String)} instead
-     * <p/>
-     * We find no use in storing the password when you can persist the {@link VimeoAccount} across
-     * application sessions.
-     */
-    @Deprecated
-    public void saveAccount(@Nullable VimeoAccount vimeoAccount, String email, String password) {
-        saveAccount(vimeoAccount, email);
-    }
-
-    /**
      * Sets the {@link #mVimeoAccount} field as well as triggering the saveAccount event for the
      * account store
      */
     public void saveAccount(@Nullable VimeoAccount vimeoAccount, String email) {
```

Comment:
Again, why ignore email in the annotations? Also, it seems a little strange to allow a null account to be passed in here. It's not really sane behavior for a caller to try to save a null account. I get that you are protecting against null below. It's also inconsistent with the rest of the annotations that you've got for saveAccount in other places.